### PR TITLE
Added chat message when hitting grilles

### LINKED ISSUE #18656: There is no attack message when you hit a grille with whatever
There is no attack message when you hit a grille with literally anything. I think it is this way because we didn't have spam filter thing in the past but I think we should add attack messages to hitting grilles now. Mainly because if it is your first time doing it with a strong hitting item, you probably break the grille before you go "huh there is no message is this thing breakable?" but when you try it for the first time and what you have is a low damage item you either think that it doesn't break or there is a bug.

### PR DESCRIPTION
**Pull request by DamianX**
Fixes #18656

### CODE REVIEW

**Comment by Kurfursten** [comment]
This reminds me, grille/broken does not call ..() in New()

line 265

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -142,6 +142,7 @@
 	return 0
 
 /obj/structure/grille/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	visible_message("<span class='danger'>[user] hits [src] with [W].</span>")
```
Comment (Exxion) [inline code/game/objects/structures/grille.dm:145]: Do we still care about `\the`? I honestly don't know

Comment (DamianX) [reply code/game/objects/structures/grille.dm:145]: I think we care in case it makes a difference. Here it wouldn't.

Comment (Exxion) [reply code/game/objects/structures/grille.dm:145]: There are no cases in which it makes a difference
It is always extraneous, without exception

Comment (Exxion) [reply code/game/objects/structures/grille.dm:145]: Actually if I don't give a shit and you don't give a shit that's already 50% of active collabs

Comment (Exxion) [reply code/game/objects/structures/grille.dm:145]: Apparently it *does* make a difference in certain rare cases which I could have sworn I had already tested

**Pull request merged**